options/locale/locale_en-US.ini

```
@@ -955,6 +955,7 @@ template.one_item = Must select at least one template item
 template.invalid = Must select a template repository
 
 archive.title = This repo is archived. You can view files and clone it, but cannot push or open issues/pull-requests.
+archive.titleDate = This repository has been archived on %s. You can view files and clone it, but cannot push or open issues/pull-requests.
```

We use snake_case for translation keys.
```suggestion
archive.repo_hint = This repository has been archived on %s. You cannot push, interact with issues, or open pull requests anymore.
```